frontends/main/src/app-pages/DashboardPage/CoursewareDisplay/EnrollmentDisplay.tsx

```diff
@@ -107,23 +175,17 @@ const EnrollmentDisplay = () => {
    * The above TODO could be handled then.
    */
   const { ended, started, notStarted } = sortEnrollments(enrolledCourses || [])
-  const sorted = [...started, ...notStarted, ...ended]
+  const shownEnrollments = [...started, ...notStarted]
```

Comment:
My understanding of the sorting and collapse logic is that it should be:

1. Started
2. Not started
3. Course over, user "Completed = Certificate Earned"
4. other cases where course is over

1–3 should be always visible, and 4 should be hidden by default.

@sovsey Can you confirm?

Assuming ☝️ is correct, we'll need to modify `sortEnrollments`... right now it doesn't distinguish between cases (3) and (4)

Reply:
@gumaerc Separately, I think the term "Completed" is pretty confusing. I would be 100% in favor of renaming `EnrollmentStatus.Completed` to `EnrollmentStatus.CertificateEarned`.